bootstrap/mac.sh

```diff
@@ -6,3 +6,13 @@ fi
 
 brew install augeas
 brew install dialog
+
+if ! hash pip 2>/dev/null; then
+    echo "pip Not Installed\nInstalling python from Homebrew..."
+    brew install python
+fi
+
+if ! hash virtualenv 2>/dev/null; then
+    echo "virtualenv Not Installed\nInstalling with pip"
+    pip install virtualenv
```

Comment:
This pip step. brew got uppity when I ran the whole script with sudo (rightly so).